src/components/postComponent/components/fileInput/filedropzone.svelte

```diff
@@ -0,0 +1,136 @@
+<script lang="ts">
+	// Importing the FileDropzone component and upload restrictions
+	import { FileDropzone } from '@skeletonlabs/skeleton';
+	import { uploadRestrictions } from '../../../../static/components/index.ts';
+
+	// Importing stores for managing file upload state
+	import { files, isFileUploaded, uploadedImageUrl, isEditing, isFileSelected } from '$stores';
+
+	// Local variable to track selected files
+	let localFiles: FileList;
+
+	// Reactively update the files store when localFiles changes
+	$: if (localFiles) {
```

Comment:
Ich glaube, hier kannst du auch direkt in den store shreiben. Wenn das nicht funktionert ist das aber auch so in ordnung :)

Reply:
Nein das hatte mit dem bind nicht funktioniert, weiß auch nicht warum